lms/static/scripts/frontend_apps/components/dashboard/test/GradeStatusChip-test.js

```diff
@@ -8,10 +8,18 @@ describe('GradeStatusChip', () => {
     return mount(<GradeStatusChip grade={grade} />);
   }
 
-  [0, 0.2, 0.48, 0.77, 0.92, 1].forEach(grade => {
+  [
```

Comment:
This maybe a bit pythonic?

Reply:
A touple is perfectly fine, but in JS, you can also "unpack" dicts, which has the benefit that you can name the values in the touple without having to go to the place where they are unpacked:

```js
[
  { grade: 0, expected: '0' },
  { grade: 0.2, expected: '20' },
  { grade: 0.33330004, expected: '33.33' },
  { grade: 0.48, expected: '48' },
  { grade: 0.77, expected: '77' },
  { grade: 0.92, expected: '92' },
  { grade: 1, expected: '100' },
].forEach(({ grade, expected }) => {...});
```

But again, the scope here is small enough that this is fine. I have done it many many times.